__tests__/HTMLView-test.js

```diff
@@ -4,7 +4,7 @@ import renderer from 'react-test-renderer';
 
 import {StyleSheet, Text} from 'react-native';
 
-import HTMLView from '../HTMLView';
+import HTMLView from '../lib';
```

Comment:
Test the built sources instead of originals to also capture possible transpilation issues